bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-resource",
-  "version": "2.2.0",
+  "version": "2.1.1",
```

Comment:
Grrrr last person must have messed this up.  Bower is out of sync with package and changelog.